src/status_im2/contexts/chat/lightbox/view.cljs

```diff
@@ -77,30 +77,34 @@
-         :key-fn                            :message-id
-         :style                             {:width (+ screen-width constants/separator-width)}
-         :data                              @data
-         :render-fn                         image
-         :render-data                       {:opacity-value    (:opacity animations)
-                                             :border-value     (:border animations)
-                                             :transparent?     transparent?
-                                             :set-full-height? set-full-height?
-                                             :screen-height    screen-height
-                                             :screen-width     screen-width
-                                             :window-height    window-height
-                                             :window-width     window-width
-                                             :props            props
-                                             :curr-orientation curr-orientation}
-         :horizontal                        horizontal?
-         :inverted                          inverted?
-         :paging-enabled                    true
-         :get-item-layout                   (fn [_ index] (get-item-layout _ index item-width))
-         :viewability-config                {:view-area-coverage-percent-threshold 50
-                                             :wait-for-interaction                 true}
-         :shows-vertical-scroll-indicator   false
+        {:ref #(reset! (:flat-list-ref props) %)
+         :key-fn :message-id
+         :on-scroll (fn [e] (utils/on-scroll e item-width (:images-opacity animations) landscape?))
+         :scroll-event-throttle 8
+         :style {:width (+ screen-width constants/separator-width)}
+         :data @data
+         :render-fn image
+         :render-data {:opacity-value     (:opacity animations)
+                       :border-value      (:border animations)
+                       :full-screen-scale (:full-screen-scale animations)
+                       :images-opacity    (:images-opacity animations)
+                       :transparent?      transparent?
+                       :set-full-height?  set-full-height?
+                       :screen-height     screen-height
+                       :screen-width      screen-width
+                       :window-height     window-height
+                       :window-width      window-width
+                       :props             props
+                       :curr-orientation  curr-orientation}
+         :horizontal horizontal?
+         :inverted inverted?
+         :paging-enabled true
+         :get-item-layout (fn [_ index] (get-item-layout _ index item-width))
+         :viewability-config {:view-area-coverage-percent-threshold 50
+                              :wait-for-interaction                 true}
+         :shows-vertical-scroll-indicator false
```

Comment:
Pleae try to preserve the map alignment. The function added to  `on-scroll` may be added to the previous `let` binding, if that's the problem